<launch>
    <!-- Launch the usb_cam node -->
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
        <param name="video_device" value="/dev/video4" />
        <param name="image_width" value="1920" />
        <param name="image_height" value="1080" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap" />
    </node>

    <!-- Launch the image_view node to visualize the camera feed -->
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/usb_cam/image_raw" />
    </node>

</launch>